python/cudf/cudf/_lib/pylibcudf/hashing.pxd

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from libc.stdint cimport uint32_t, uint64_t
+from cudf._lib.cpp.types cimport size_type
+from cudf._lib.cpp.table cimport table
+from libcpp.vector cimport vector
```

Comment:
I think this is all you need here. You may have to add some cimports to the pyx file after this, though.
```suggestion
from libc.stdint cimport uint32_t, uint64_t
```